src/bttrx_wifi.cpp

```diff
@@ -164,9 +166,21 @@ void BTTRX_WIFI::setup(BTTRX_CONTROL *control)
 	}
 	bttrx_control_ = control;
 
+	Serial.println("");
 	Serial.println("Configuring access point...");
 
-	WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+	String callsign = preferences.getString("callsign", "");
```

Comment:
see comment above, please implement getter method in bttrx_control for this

Reply:
[x] done. Will be included in the PR against the dev branch.